apis/nucleus/src/components/listbox/components/ListBoxHeader/icon-utils.js

```diff
@@ -17,16 +18,25 @@ const createDimensionIconData = (dimInfo, app) => {
       };
     case dimensionTypes.cyclic:
       return {
-        icon: CyclicIcon,
+        icon: app ? ReloadIcon : CyclicIcon,
         tooltip: 'Listbox.Cyclic',
-        onClick: () => {
-          app
-            .getDimension(dimInfo.qLibraryId)
-            .then((dimensionModel) => {
-              dimensionModel.stepCycle(1);
-            })
-            .catch(() => null);
-        },
+        onClick: app
+          ? () => {
+              app
+                .getDimension(dimInfo.qLibraryId)
+                .then((dimensionModel) => {
+                  if (!dimensionModel.stepCycle) {
+                    // eslint-disable-next-line no-console
+                    console.log(
+                      "engine version or engine api spec version doesn't have support for function stepCycle"
```

Comment:
maybe remove engine version?
from what I know it is only run when engine api spec version doesn't have support for function stepCycle.
if (only) engine version don't have support for stepCycle it will not get here (only the `.catch(() => null)` later)

Reply:
I see then I will remove engine version

Reply:
fixed